src/modules/standard-clauses/standard-clauses.controller.ts

```
@@ -1,62 +1,119 @@
+import {
+  ApiBearerAuth,
+  ApiOperation,
+  ApiResponse,
+  ApiTags,
+} from '@nestjs/swagger';
 import { StandardClausesService } from './standard-clauses.service';
 import { CreateStandardClauseDto } from './dto/create-standard-clause.dto';
 import { StandardClause } from '../../entities/standard-clause.entity';
 import { JwtAuthGuard } from '../../auth/guards/jwt-auth.guard';
+import { UpdateStandardClauseDto } from './dto/update-standard-clause.dto';
+import { UpdateStandardClauseDto } from './dto/update-standard-clause.dto';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'UpdateStandardClauseDto'. Consider to delete it or rename it.

'UpdateStandardClauseDto' is defined here:


(lint/suspicious/noRedeclare)